### PR TITLE
zh:update/docs/setup/install/multicluster/before-you-begin/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
The english have change!

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/setup/install/multicluster/before-you-begin/index.md [hunk 1]
```diff
@@ -51,9 +51,30 @@ $ export CTX_CLUSTER2=<your cluster2 context>
...
+If you are planning to deploy only one primary cluster (i.e., one of the
+Primary-Remote installations, below), you will only have a single CA
+(i.e., `istiod` on `cluster1`) issuing certificates for both clusters.
+In that case, you can skip the following CA certificate generation step
+and simply use the default self-signed CA for the installation.
+{{< /tip >}}
+
+This guide will assume that you use a common root to generate intermediate
+certificates for each primary cluster.
+Follow the [instructions](/docs/tasks/security/cert-management/plugin-ca-cert/)
```
Comment (yanrongshi) [inline content/zh/docs/setup/install/multicluster/before-you-begin/index.md:64]:
```suggestion
Follow the [instructions](/zh/docs/tasks/security/cert-management/plugin-ca-cert/)
```

File: content/zh/docs/setup/install/multicluster/before-you-begin/index.md [hunk 2]
```diff
@@ -51,9 +51,30 @@ $ export CTX_CLUSTER2=<your cluster2 context>
 参阅[证书管理](/zh/docs/tasks/security/cert-management/)，以了解所有可用选项的详细描述和说明。
 根据你选择的方式，Istio 的安装说明可能略有变化。
 
-本指南假设你使用一个公共根，为每个集群生成中间证书。
-按照[说明](/zh/docs/tasks/security/cert-management/plugin-ca-cert/)，
-生成并分别推送 CA 证书 secret 给 `cluster1` 和 `cluster2`。
+<!-- {{< tip >}}
```
Comment (kebe7jun) [inline content/zh/docs/setup/install/multicluster/before-you-begin/index.md:54]: 注释掉的这部分我建议拿掉。

Comment (yanrongshi) [reply content/zh/docs/setup/install/multicluster/before-you-begin/index.md:54]: 收到，已更新

**Review by zirain — APPROVED** [review]

**Pull request merged**